resources/conf/config.toml.template

```diff
@@ -63,6 +63,26 @@ artifactsDirectory = "/home/wso2/artifacts"
   # Optional path to the private key for Consul communication. If this is set, then you need to also set certFile
   keyFile = "/home/wso2/security/truststore/consul/local-dc-client-consul-0-key.pem"
 
+# Configuration related to the repository where the API artifacts are stored
+[adapter.sourceControl]
+  # Enable/Disable Source Control for API Artifacts
+  enabled = true
```

Comment:
The configs have to be added only if the feature is being used.

Reply:
It seems in other configs we are keeping the enable value as `false` if feature is not enabled by-default.

Reply:
The default value of enabled is set to false in default_config.go

Reply:
Fixed with 763b649

Reply:
I too think that it would be better to set this to false. Looks like we have set all other values to be same as what we have set in `default_config.go`

Reply:
In that case it should be set to false. (config.toml.template represents the default config's state)